src/test/scala/uk/gov/nationalarchives/LambdaTest.scala

```diff
@@ -230,30 +264,41 @@ class LambdaTest extends AnyFlatSpec with BeforeAndAfterEach {
   "handleRequest" should "copy the correct child assets from source to destination" in {
     stubGetRequest(dynamoGetResponse)
     stubQueryRequest(dynamoQueryResponse)
-    val source = s"/$batchId/data/$childId"
-    val destination = s"/opex/$executionName/$assetParentPath/$assetId.pax/Representation_Preservation/${childId}/Generation_1/$childId.json"
-    stubCopyRequest(source, destination)
+    val sourceJson = s"/$batchId/data/$childIdJson"
+    val destinationJson = s"/opex/$executionName/$assetParentPath/$assetId.pax/Representation_Preservation/$childIdJson/Generation_1/$childIdJson.json"
+    stubCopyRequest(sourceJson, destinationJson)
+    val sourceDocx = s"/$batchId/data/$childIdDocx"
+    val destinationDocx = s"/opex/$executionName/$assetParentPath/$assetId.pax/Representation_Preservation/$childIdDocx/Generation_1/$childIdDocx.docx"
+    stubCopyRequest(sourceDocx, destinationDocx)
     val xipPath = s"/opex/$executionName/$assetParentPath/$assetId.pax/$assetId.xip"
     val opexPath = s"/opex/$executionName/$assetParentPath/$assetId.pax.opex"
     stubPutRequest(xipPath, opexPath)
 
     TestLambda().handleRequest(standardInput, outputStream, null)
 
-    val s3CopyRequest = s3Server.getAllServeEvents.asScala.filter(_.getRequest.getUrl == destination).head.getRequest
-    s3CopyRequest.getUrl should equal(destination)
-    s3CopyRequest.getHost should equal("test-destination-bucket.localhost")
-    s3CopyRequest.getHeader("x-amz-copy-source") should equal(s"test-source-bucket$source")
+    def checkCopyRequest(source: String, destination: String) = {
+      val s3CopyRequest = s3Server.getAllServeEvents.asScala.filter(_.getRequest.getUrl == destination).head.getRequest
+      s3CopyRequest.getUrl should equal(destination)
+      s3CopyRequest.getHost should equal("test-destination-bucket.localhost")
+      s3CopyRequest.getHeader("x-amz-copy-source") should equal(s"test-source-bucket$source")
```

Comment:
Should this test also check what's in the body of the request?

Reply:
There's nothing in the body of a copy request. It just comes back as empty string so it's not really worth it.